x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/create_engine_flyout.tsx

```diff
@@ -127,44 +127,56 @@ export const CreateEngineFlyout = ({ onClose }: CreateEngineFlyoutProps) => {
         )}
       </EuiFlyoutHeader>
       <EuiFlyoutBody>
-        <EuiSteps
-          steps={[
-            {
-              children: (
-                <IndicesSelectComboBox
-                  fullWidth
-                  isDisabled={formDisabled}
-                  onChange={onIndicesChange}
-                  selectedOptions={selectedIndices.map((index: string) => indexToOption(index))}
-                />
-              ),
-              status: indicesStatus,
-              title: i18n.translate(
-                'xpack.enterpriseSearch.content.engines.createEngine.selectIndices.title',
-                { defaultMessage: 'Select indices' }
-              ),
-            },
-            {
-              children: (
-                <EuiFieldText
-                  fullWidth
-                  disabled={formDisabled}
-                  placeholder={i18n.translate(
-                    'xpack.enterpriseSearch.content.engines.createEngine.nameEngine.placeholder',
+        <EuiFlexGroup direction="column">
+          <EuiFlexItem grow>
+            <EuiCallOut title="Technical Preview feature" color="warning" iconType="beaker">
+              <FormattedMessage
+                id="xpack.enterpriseSearch.content.engines.createEngine.technicalPreviewCallOut.title"
+                defaultMessage="This functionality is in technical preview and may be changed or removed completely in a future release. Elastic will take a best effort approach to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."
```

Comment:
Can we quickly change this to say:

> This functionality may be changed or removed completely in a future release.

To make it less wordy

Reply:
Just realized this was in the wrong place, need this for the _popover_, not the callout. Callout is fine as is.